doc/configuration.rst

```diff
@@ -1347,8 +1363,7 @@ OpenOCDDriver
 An OpenOCDDriver controls OpenOCD to bootstrap a target with a bootloader.
 
 Note that OpenOCD supports specifying USB paths since
-`a1b308ab <https://sourceforge.net/p/openocd/code/ci/a1b308ab/>`_ which is not
-part of a release yet.
+`a1b308ab <https://sourceforge.net/p/openocd/code/ci/a1b308ab/>`_ which was released with v0.11.
```

Comment:
Could you update the message in `OpenOCDDriver._get_usb_path_cmd()` accordingly?

Reply:
I've updated the message in the driver as well.